sdks/java/io/google-cloud-platform/build.gradle

```diff
@@ -31,17 +31,20 @@ ext.summary = "IO library to read and write Google Cloud Platform systems from B
 dependencies {
   compile enforcedPlatform(library.java.google_cloud_platform_libraries_bom)
   compile project(path: ":sdks:java:core", configuration: "shadow")
-  compile project(":sdks:java:expansion-service")
+  testCompile project(":sdks:java:expansion-service")
```

Comment:
@ibzib does your comment about making the :sdks:java:expansion-service dependency runtimeOnly apply here too?

Reply:
This one is actually needed at compile time, which is why it passes the strict dependency checker: https://github.com/apache/beam/blob/0c01636fc8610414859d946cb93eabc904123fc8/sdks/java/io/google-cloud-platform/src/test/java/org/apache/beam/sdk/io/gcp/pubsub/PubsubIOExternalTest.java#L32

Reply:
> Run Python 3.8 PostCommit

@ibzib I see the error
Error: Could not find or load main class org.apache.beam.sdk.expansion.service.ExpansionService
Changing to runtimeOnly.